### PR TITLE
Rack Elevation Endpoint

### PR DESCRIPTION
**Pull request by minitriga**
Fixes issue #237 

However, not sure if a new endpoint is required or deleting the old one?

Potentially checking version for greater than 2.8?

### CODE REVIEW

File: pynetbox/models/dcim.py [hunk 1]
```diff
@@ -119,6 +119,23 @@ def units(self):
...
+        Returns a DetailEndpoint object that is the interface for
+        viewing response from the elevation endpoint updated in Netbox version 2.8.
+
+        :returns: :py:class:`.DetailEndpoint`
+
+        :Examples:
+
+        >>> rack = nb.dcim.racks.get(123)
+        >>> rack.elevation.list()
+        {"get_facts": {"interface_list": ["ge-0/0/0"]}}
```
Comment (zachmoody) [inline pynetbox/models/dcim.py:135]: Quick nit (I just realized I have a typo in the `units()` docs too :man_facepalming:), but could you add a quick sample output here? I don't have a 2.8 instance readily available.

File: tests/test_dcim.py [hunk 2]
```diff
@@ -401,6 +401,28 @@ def test_get_units(self, mock):
             isinstance(ret[0].device, pynetbox.models.dcim.Devices)
         )
 
+    @patch(
+        'pynetbox.core.query.requests.sessions.Session.get',
+        side_effect=[
+            Response(fixture='dcim/rack.json'),
+            Response(fixture='dcim/rack_u.json'),
```
Comment (zachmoody) [inline tests/test_dcim.py:408]: Is it really the same return as `units`?

Comment (minitriga) [reply tests/test_dcim.py:408]: Yes just got this from netboxdemo.com

```

{
    "count": 12,
    "next": null,
    "previous": null,
    "results": [
        {
            "id": 12,
            "name": "U12",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": {
                "id": 16,
                "url": "https://netboxdemo.com/api/dcim/devices/16/",
                "name": "DS9_SECOFFICE_FIBER_U12",
                "display_name": "DS9_SECOFFICE_FIBER_U12"
            }
        },
        {
            "id": 11,
            "name": "U11",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": {
                "id": 20,
                "url": "https://netboxdemo.com/api/dcim/devices/20/",
                "name": "DS9_SECOFFICE_U11",
                "display_name": "DS9_SECOFFICE_U11"
            }
        },
        {
            "id": 10,
            "name": "U10",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": null
        },
        {
            "id": 9,
            "name": "U9",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": null
        },
        {
            "id": 8,
            "name": "U8",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": null
        },
        {
            "id": 7,
            "name": "U7",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": null
        },
        {
            "id": 6,
            "name": "U6",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": null
        },
        {
            "id": 5,
            "name": "U5",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": {
                "id": 26,
                "url": "https://netboxdemo.com/api/dcim/devices/26/",
                "name": "DS9_SECOFFICE_ACCESS",
                "display_name": "DS9_SECOFFICE_ACCESS"
            }
        },
        {
            "id": 4,
            "name": "U4",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": null
        },
        {
            "id": 3,
            "name": "U3",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": null
        },
        {
            "id": 2,
            "name": "U2",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": null
        },
        {
            "id": 1,
            "name": "U1",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": {
                "id": 18,
                "url": "https://netboxdemo.com/api/dcim/devices/18/",
                "name": "DS9_SECOFFICE_PDU",
                "display_name": "DS9_SECOFFICE_PDU"
            }
        }
    ]
}
```

Comment (zachmoody) [reply tests/test_dcim.py:408]: oh I see, didn't know units was replaced. This works then. Sorry for the confusion.

**Pull request merged**